src/menu/settings_menu_template.js

```diff
@@ -12,6 +12,15 @@ export const settingsMenu = {
         item.checked = autoHideMenuPref;
         window.setAutoHideMenuBar(autoHideMenuPref);
       }
+    },
+    {
+      label: "Start In Tray",
```

Comment:
Just a consistency thing: this label doesn't say "Toggle" at the beginning, while the Auto Hide Menu Bar says "Toggle Auto Hide Menu Bar". I'm in favor of removing the "Toggle" wording from that one since the checkbox seems to make the toggle behavior clear.

Any input, @chrisknepper?

Reply:
Agreed!